charts/minikube-values.yaml

```diff
@@ -22,3 +22,7 @@ jupyterhub:
     state:
       enabled: true
       cryptoKey: 86282f03d9886a64a46ee38f946e9ee27a600df4559584eeb90d0bfbd5a3dc0e
+
+notebooks:
+  jupyterhub_base_url: /jupyterhub/
+  jupyterhub_api_token: notebookstoken
```

Comment:
`\n\0`

Reply:
I have no idea why this newline is not showing up -- it's there in my editor and also in the online github editor...

Reply:
it's fine now